modules/actionagent/step_store.go

```diff
@@ -70,6 +71,10 @@ func (agent *Agent) store() {
 }
 
 func (agent *Agent) storeCache(tarFile, cachePath string) (err error) {
+	if cachePathSize, isExceed := agent.isCachePathExceedLimit(cachePath); isExceed {
+		return fmt.Errorf("tar path: %s size: %d b exceed limit size: %d b", cachePath, cachePathSize,
```

Comment:
no err, just warn log

Reply:
just log and continue try next cache file
![image](https://user-images.githubusercontent.com/30427474/149305076-ec7daeb7-67e1-494d-808e-6fcfdcc49ee1.png)